scripts/check_frontend_coverage_test.py

```diff
@@ -308,14 +291,19 @@ def test_fully_covered_filenames_is_not_sorted(self):
         )
         fully_covered_tests_swap = self.swap(check_frontend_coverage,
             'fully_covered_filenames', [
-                'anotherfile.ts'
                 'file.ts',
-            ])
+                'anotherfile.ts'
+            ]
+        )
 
         with self.exists_swap, self.open_file_swap, self.print_swap:
             with fully_covered_tests_swap:
-                self.assertEqual(
-                    self.check_function_calls_counter['mock_print'], 0)
+                with self.assertRaises(SystemExit) as error:
+                    (check_frontend_coverage
+                        .check_fully_covered_filenames_list_is_sorted())
+                self.assertEqual(error.exception.message,
```

Comment:
Throughout: prefer assertRaisesRegexp instead of assertRaises.

Reply:
Done.